public/assets/styles/modules/validation.css

```diff
@@ -0,0 +1,73 @@
+.mention {
+  align-self: end;
+  margin-top: 2em;
+  margin-bottom: -1em;
+}
+
+.mention::before {
+  margin-right: .4em;
+}
+
+.mention::before, .requis::before {
+  content: "*";
+  color: var(--rose-anssi);
+}
+
+.requis::before {
+  position: absolute;
+  left: -1em;
+}
+
+.requis {
+  position: relative;
+}
+
+.message-erreur {
+  display: none;
+
+  margin-top: 1em;
+
+  color: var(--rose-anssi);
+  font-weight: normal;
+  font-size: 0.85em;
+}
+
+.requis[data-nom = 'cguAcceptees'] {
+  margin-bottom: 2em;
+}
+
+:is(input, select):valid{
+  border-color: var(--bleu-mise-en-avant);
+}
+
+input[type="radio"]:valid {
+  background-image: url('data:image/svg+xml;utf8,<svg viewBox="0 0 100 100" xmlns="http://www.w3.org/2000/svg"><circle cx="50" cy="50" r="45" style="fill: white; stroke: %230f7ac7; stroke-width: 0.50em" /></svg>')
+}
+
+:is(input, select):invalid {
```

Comment:
On avait évoqué l'idée d'être plus robustes ici en utilisant un sélecteur plus précis : `:is(input, select):not(.intouche):invalid`